package.json

```diff
@@ -12,6 +12,8 @@
         "access": "public"
     },
     "scripts": {
+        "predev": "tsx scripts/create-icon-type-names-definition.ts",
```

Comment:
script that generates the icon name type from the icons.svg file. runs on dev and build